package.json

```diff
@@ -1,26 +1,27 @@
 {
-  "name": "project-oregano",
+  "name": "website-v2",
   "private": true,
-  "description": "Official repository for HackNITR 3.0",
+  "description": "Official repository for TEDxNITRourkela",
   "version": "0.1.0",
   "license": "MIT",
```

Comment:
I don't think we are using MIT License. Confirm once from @rutajdash regarding this, he knows the specifics.

Reply:
put "SEE LICENSE IN LICENSE.md"